_posts/en/newsletters/2023-12-13-newsletter.md

```diff
@@ -0,0 +1,202 @@
+---
+title: 'Bitcoin Optech Newsletter #281'
+permalink: /en/newsletters/2023/12/13/
+name: 2023-12-13-newsletter
+slug: 2023-12-13-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Discussion about griefing liquidity ads:** Bastien Teinturier
```

Comment:
Totally optional, but readers could be interested in some our chat with Lisa/tbast a couple weeks ago? (could just be a link). https://bitcoinops.org/en/podcast/2023/11/30/#update-to-the-liquidity-ads-specification-transcript